packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -31,7 +31,7 @@ export function useNotifyCopy() {
 		let notice = '';
 		if ( selectedBlockClientIds.length === 1 ) {
 			const clientId = selectedBlockClientIds[ 0 ];
-			const { title } = getBlockType( getBlockName( clientId ) );
+			const { title } = getBlockType( getBlockName( clientId ) ) || null;
```

Comment:
Destructuring null will still cause an error. How about:
```
const title = getBlockType( getBlockName( clientId ) )?.title;
```

<img width="597" alt="CleanShot 2021-09-23 at 13 06 19@2x" src="https://user-images.githubusercontent.com/1270189/134576304-ebe133b4-7c4d-4f7f-9945-41f9bcaf93d5.png">

Reply:
@gwwar, my bad. I didn't notice this change. This is not my final change. Because I tested this thing for other places. My dev environment (Github codespaces) stopped working abruptly 😩  Not sure why! Few changes might get pushed which were not part of the final changes. Let me fix it.